web/multinode/src/nodes/index.ts

```diff
@@ -49,7 +51,7 @@ export class Node {
     }
 
     public get statusText(): string {
-        return NodeStatus[this.status];
+        return this.status;
```

Comment:
Why this change?

Reply:
@ifraixedes when we are fetching nodes we are already setting the status to `NodeStatus[node.status]`.
```
 return nodeListJson.map(node => new Node(
            node.id,
            node.name,
            node.version,
            new Date(node.lastContact),
            node.diskSpaceUsed,
            node.diskSpaceLeft,
            node.bandwidthUsed,
            0,
            0,
            0,
            node.totalEarned,
            NodeStatus[node.status],
        ));```
say if the node status is `online` from the api then in the node object this.status become `NodeStatus["online"]` which is `Online`.  So for `NodeStatus[this.status]` it will give `NodeStatus['Online']` which results `undefined`.

Reply:
I'm not much into the frontend, but in your above code snipped, it uses `node.status`, no `node.statusText()`

Reply:
It actually uses 
```export enum NodeStatus {
    'online' = 'Online',
    'offline' = 'Offline',
    'not reachable' = 'Not reachable',
    'unauthorized' = 'Unauthorized',
    'storagenode internal error' = 'Internal Error'
}``` 
This enum. where status is the key and statusText is the value. That's why I think it was named like this. 
Without the change the statusText will return NodeStatus['Online'] which is undefined.

Reply:
It returns the status on the node according to NodeStatus enum values. The keys are all in lowercase but the values are capitalized.

Reply:
@ptdatta is right. In the constructor, the `this.status` is already set to `NodeStatus[node.status]`. Calling that again in `statusText` will most likely return an empty string.

Reply:
Yes it's returning `undefined`.

Reply:
My question was "What is calling this method". If it turns out that nothing calls it, could we remove it?